extensions/schema-compare/package.json

```diff
@@ -1,59 +1,64 @@
 {
-  "name": "schema-compare",
-  "displayName": "SQL Server Schema Compare",
-  "description": "SQL Server Schema Compare for Azure Data Studio supports comparing the schemas of databases and dacpacs.",
-  "version": "0.2.0",
-  "publisher": "Microsoft",
-  "preview": true,
-  "engines": {
-    "vscode": "^1.25.0",
-    "sqlops": "*"
-  },
-  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/extensions/import/Microsoft_SQL_Server_Import_Extension_and_Tools_Import_Flat_File_Preview.docx",
-  "icon": "images/sqlserver.png",
-  "aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
-  "activationEvents": [
-    "*"
-  ],
-  "main": "./out/main",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Microsoft/azuredatastudio.git"
-  },
-  "extensionDependencies": [
-    "Microsoft.mssql"
-  ],
-  "contributes": {
-    "commands": [
-      {
-        "command": "schemaCompare.start",
-        "title": "Schema Compare",
-        "icon": {
-          "light": "./images/light_icon.svg",
-          "dark": "./images/dark_icon.svg"
-        }
-      }
-    ],
-    "menus": {
-      "objectExplorer/item/context": [
-        {
-          "command": "schemaCompare.start",
-          "when": "connectionProvider == MSSQL && nodeType && nodeType == Database",
-          "group": "export"
-        }
-      ]
-    }
-  },
-  "dependencies": {
-    "vscode-extension-telemetry": "0.0.18",
-    "vscode-nls": "^3.2.1"
-  },
-  "devDependencies": {
-    "@types/mocha": "^5.2.5",
+	"name": "schema-compare",
```

Comment:
do we know why this file keeps getting updated in unrelated PRs?  This the second time I've seen this and the changes appear to contain more than just reformatting.

Reply:
actually, i wasn't aware of the formatting changes until i see it here. the change was for the __metadata section added by the build step, i didn't look into why, but seems inherited from vscode. all other extensions had been updated with this section.